app/Event.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Event extends Model
+{
+
+    /**
+     * The attributes that are not mass assignable.
+     *
+     * @var array
+     */
+    protected $guarded = [
+        'id', 'created_at', 'updated_at'
+    ];
+
+    public function organizer()
+    {
+        return $this->hasOne('App\User');
+    }
+
+    public function rsvp()
+    {
+        return $this->belongsTo('App\Rsvp');
```

Comment:
(discussed in Slack) should be a hasMany()

Reply:
fixed